docs/en/observability/whats-new.asciidoc

```diff
@@ -128,4 +128,49 @@ Elastic Observability. For a holistic view of your deployments, you can also qui
 [role="screenshot"]
 image::images/7.14-cluster-id-kubernetes.png[Kubernetes metrics]
 
+[discrete]
+== Histogram support for OpenTelemetry agents
+
+Histogram support for OpenTelemetry agents enables you to profile the distribution of measurements for user analytics and troubleshooting.
+You can now identify subpopulation patterns in your traffic with granular analysis of OpenTelemetry time series data, such as transaction latency, 
+payload size, and business KPIs.
+
+[discrete]
+== APM transaction correlations (beta)
+
+As it is virtually impossible to manually analyze what has become an overwhelming amount of data and dependencies in modern applications,
+automation and analytics have become essential components of any troubleshooting toolkit. Elastic’s APM correlations feature (beta)
+accelerates root cause analysis by automatically surfacing attributes of your data set (such as infrastructure components, versions, and
+locations) that are correlated with high-latency or erroneous transactions and have the most significant impact on your service performance.
+You can visualize the latency distribution of any attribute compared to overall latency and use these attributes to filter and isolate the root
+causes of your performance problems.
+
+[role="screenshot"]
+image::images/apm-correlations.png[APM correlations]
```

Comment:
Looks like the doc build is failing because some images are missing.

Reply:
Ah, good catch. I redid my changes from a clean branch and forgot to stage the images too.